app/src/main/java/org/kiwix/kiwixmobile/main/KiwixWebViewClient.java

```diff
@@ -106,11 +112,34 @@ public void onPageFinished(WebView view, String url) {
 
   private void inflateHomeView(WebView view) {
     LayoutInflater inflater = LayoutInflater.from(view.getContext());
-    home = inflater.inflate(R.layout.content_main, view, false);
+    if (KiwixApplication.getInstance().getResources().getConfiguration().orientation ==
+        Configuration.ORIENTATION_PORTRAIT) {
+      home = inflater.inflate(R.layout.content_main_p, view, false);
+    } else if (KiwixApplication.getInstance().getResources().getConfiguration().orientation ==
+        Configuration.ORIENTATION_LANDSCAPE) {
+      home = inflater.inflate(R.layout.content_main_l, view, false);
+    }
     callback.setHomePage(home);
     if (sharedPreferenceUtil.nightMode()) {
       ImageView cardImage = home.findViewById(R.id.content_main_card_image);
-      cardImage.setImageResource(R.drawable.ic_home_kiwix_banner_night);
+      ImageView sideImage = home.findViewById(R.id.content_side_image);
+      AppCompatButton downloadButton = home.findViewById(R.id.content_main_card_download_button);
+      downloadButton.setTextColor(Color.parseColor("#000000"));
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        downloadButton.setBackgroundTintList(ColorStateList.valueOf(
```

Comment:
You can do this using Butter Knife. You can refer to http://jakewharton.github.io/butterknife/ and find **R.color** on this webpage to search how to inject color in an instance variable.

Reply:
I tried doing using Butterknife but it is throwing an error " Resource not found exception" so I did this.